docs/contributing/managing-change.md

```diff
@@ -0,0 +1,171 @@
+# Managing change
+
+When making changes to the codebase that could affect our users, we want to minimise the number of breaking changes in a major version that cannot be made before updating to the major version.
```

Comment:
Slightly unsure what we mean by the second half of this sentence, "...in a major version that cannot be made before updating to the major version". Are we talking about minimising the number of breaking changes that a user might be unable to make in their own service/port?

Reply:
We want as many breaking changes as possible to be 'actionable' in minor releases that come prior to the major release.

So if a user has followed the release notes by the time they upgrade to the next major version there should be few changes they actually have to make at that point.

Does that make sense? Agree it's clumsily worded at the minute – maybe @claireashworth can help?

Reply:
How about something like this:

Breaking changes should, wherever possible, be actionable by the users. By doing this the users will have fewer actions to do to complete a major release.

Reply:
Ah I see, thanks for explaining. So it's more about, where possible, telling users about changes we're making in patch/feature releases so they're given a heads up earlier and are given the opportunity to make changes to their service in advance to prepare?

Reply:
Exactly that 👍🏻

Reply:
@claireashworth I think the missing bit is that the changes should ideally be actionable in prior minor or patch versions.

That's not always going to be possible but we should aim to do it where it makes sense. The rest of the doc mostly provides strategies for doing so.

Reply:
Could we say something like:

```Where possible, users should be notified about upcoming breaking changes in a patch or feature release. By doing this, users have the opportunity to make the relevant changes in advance, so they have fewer actions to complete when we do a major release.```

Reply:
I've pushed a change that tries to simplify the 'intro' down to the bare minimum, and let the other sections go in to more detail on how it helps.

Thoughts?

Reply:
The intro looks good. A couple of style guide things - change 'whilst' to 'while' and expand 'don't' (negative verbs are written in full for gov.uk style)